<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>反汇编窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser_disa"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>日志窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser_log"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>内存窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser_memory"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>寄存器窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser_register"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_OpenFile"/>
    <addaction name="action_Attach"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>调试</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>设置断点</string>
     </property>
     <addaction name="action_SoftBreakpoint"/>
     <addaction name="action_HardBreakpoint"/>
     <addaction name="action_MemoryBreakpoint"/>
    </widget>
    <addaction name="action_run"/>
    <addaction name="separator"/>
    <addaction name="action_detach"/>
    <addaction name="menu_3"/>
    <addaction name="action_ShowMemory"/>
    <addaction name="action_Register"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_showEvent"/>
    <addaction name="action_showModule"/>
    <addaction name="action_transAddress"/>
    <addaction name="action_clear"/>
    <addaction name="action_getAddress"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_OpenFile">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Attach">
   <property name="text">
    <string>附加</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_run">
   <property name="text">
    <string>运行</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_detach">
   <property name="text">
    <string>解除附加</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_CancelBreakpoint">
   <property name="text">
    <string>取消断点</string>
   </property>
  </action>
  <action name="action_ShowMemory">
   <property name="text">
    <string>显示内存数据</string>
   </property>
  </action>
  <action name="action_Register">
   <property name="text">
    <string>显示寄存器</string>
   </property>
  </action>
  <action name="action_SoftBreakpoint">
   <property name="text">
    <string>软件断点</string>
   </property>
  </action>
  <action name="action_HardBreakpoint">
   <property name="text">
    <string>硬件断点</string>
   </property>
  </action>
  <action name="action_MemoryBreakpoint">
   <property name="text">
    <string>内存断点</string>
   </property>
  </action>
  <action name="action_showEvent">
   <property name="text">
    <string>调试事件类型</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_showModule">
   <property name="text">
    <string>载入的模块</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_transAddress">
   <property name="text">
    <string>地址转换</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="text">
    <string>清除日志窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_getAddress">
   <property name="text">
    <string>地址获取</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
